docs/content/installation/installing-nic/installation-with-manifests.md

```diff
@@ -57,41 +56,73 @@ This guide assumes you are using the latest release.
 ## Create custom resources {#create-custom-resources}
 
 To make sure your NGINX Ingress Controller pods reach the `Ready` state, you'll need to create custom resource definitions (CRDs) for various components. Alternatively, you can disable this requirement by setting the `-enable-custom-resources` command-line argument to `false`.
+There are two ways you can install the custom resource definitions:
+   1. Using a URL to apply a single CRD yaml file. We recommend this approach.
+   2. Applying your local copy of the CRD yaml files. This requires that you [clone the repository](clone-the-repository)
+
+{{<tabs name="install-crds">}}
+
+{{%tab name="Install CRDs from single YAML"%}}
 
 ### Core custom resource definitions
 
-1. Create CRDs for [VirtualServer and VirtualServerRoute]({{< relref "configuration/virtualserver-and-virtualserverroute-resources.md" >}}), [TransportServer]({{< relref "configuration/transportserver-resource.md" >}}), and [Policy]({{< relref "configuration/policy-resource.md" >}}):
+1. Create CRDs for [VirtualServer and VirtualServerRoute]({{< relref "" >}}), [TransportServer]({{< relref "configuration/transportserver-resource.md" >}}), [Policy]({{< relref "configuration/policy-resource.md" >}}) and [GlobalConfiguration]({{< relref "configuration/global-configuration/globalconfiguration-resource.md" >}}):
 
     ```shell
-    kubectl apply -f common/crds/k8s.nginx.org_virtualservers.yaml
-    kubectl apply -f common/crds/k8s.nginx.org_virtualserverroutes.yaml
-    kubectl apply -f common/crds/k8s.nginx.org_transportservers.yaml
-    kubectl apply -f common/crds/k8s.nginx.org_policies.yaml
+    kubectl apply -f https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v3.3.2/deploy/crds.yaml
```

Comment:
qq @shaun-nx @pdabelf5 now that we're pointing the user to a specific release version, in `main` should this point to `main` or `3.4.0` or `3.3.2` (and we assume that release scripts will up the version during release process)?

Reply:
@vepatel in my mind, this should be `3.3.2` right now as that's the current release.
I just ran the script under `.github/scripts` and confirmed that these version numbers are updated.